docs/en/changelogs/2.3.0.md

```diff
@@ -29,7 +29,7 @@ In order to do this, `SiteTree->ViewersGroup` and `SiteTree->EditorsGroup` have
 to many_many relations.
 
 Your group-assignements for "Who can view this page" and "Who can edit this page" should be automatically migrated upon
-calling `dev/build`. See [#2847](http://open.silverstripe.com/ticket/2847)
+calling `dev/build`. See `[#2847](http://open.silverstripe.com/ticket/2847)`
```

Comment:
GH parser parses this as literal code block and not into a link it would need to be:

```
[`#2847`](http://open.silverstripe.com/ticket/2847)
```